### PR TITLE
Add data ID tags for trafficlight adapter to find

### PR DESCRIPTION
**Pull request by michaelkaye**
This PR adds an initial set of data-testid tags to enable certain elements on the page to be found via playwright's lookup mechanisms. 

It also includes a quick change for the issue #978, which i was otherwise running into each time I built. I should get rid of that commit and maybe put it in another branch somewhere, but would appreciate guidance on what to do with it properly - this change seems to JustWork for me, so i don't know why we wouldn't want to include it.

### CODE REVIEW

**Comment by michaelkaye** [comment]
I should also mention there will be further sets of changes incoming; but this is a short set that will help for now.

**Review by SimonBrandner — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "type": "module",
   "scripts": {
     "dev": "vite",
-    "build": "vite build",
+    "build": "NODE_OPTIONS=--max-old-space-size=16384 vite build",
```
Comment (dbkr) [inline package.json:6]: We should remove this from the github action if we're putting it in here

Comment (michaelkaye) [reply package.json:6]: Removed these in 9c8692d

**Pull request merged**